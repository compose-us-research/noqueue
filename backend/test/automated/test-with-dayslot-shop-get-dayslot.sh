#!/bin/bash

source setup-dayslot.sh
testIt "Dayslot should look same" <(curl -s -w "\n___HEADER___\n%{http_code}\n" $BASE_URL/shop/$SHOP_NAME/dayslot/ -H "accept: application/json") test-with-dayslot-shop-get-dayslot.expected
