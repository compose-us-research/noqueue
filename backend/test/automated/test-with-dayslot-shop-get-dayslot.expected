{"member":[{"id":1,"customers":5,"start":"2020-11-20T00:00:00.000Z","end":"2020-11-23T00:00:00.000Z","maxDuration":2,"minDuration":1},{"id":2,"customers":3,"start":"2020-11-24T00:00:00.000Z","end":"2020-11-28T00:00:00.000Z","maxDuration":3,"minDuration":1}]}
___HEADER___
200
